{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -46,9 +47,12 @@ jobs:
           os: [macOS-latest, ubuntu-latest, windows-latest]
           python-version: {% raw %}${{ fromJSON(needs.environment-config.outputs.python-matrix) }}{% endraw %}
           mdanalysis-version: ["latest", "develop"]
+          exclude:
+            - mdanalysis-version: "latest"
```

Comment:
This won't work.. the conda-forge install step doesn't work - it shouldn't attempt to install MDAnalysis first